### ISSUE TITLE
autodoc: a more compact way to document static members

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#431]
A fairly coding style is to put comments on the same line as variable/attribute definition:

```
#!python
class Bits:
    SOFT_RESET = 0x01 #: comment for SOFT
    HARD_RESET = 0x02 #: comment for HARD
```

The major advantage of this style is its compactness.

I think it would be very useful to support this style in autodoc? (Currently autodoc wants special comments to be put on a separate line, which essentially doubles the size of the code)

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/431
- Originally reported by: Anonymous
- Originally created at: 2010-05-27T23:00:09.033

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-15 14:43:09+00:00_

Implemented in <<cset a63230f12d1f5af6de24a59ba12c17cc8943cdcd>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:22+00:00_

Removing version: 0.6.5 (automated comment)
